alephium/contracts/governance.ral

```diff
@@ -92,22 +92,22 @@ Contract Governance(
     }
 
     @using(readonly = true, externalCallCheck = false)
-    pub fn parseAndVerifyGovernanceVAAGeneric(vaa: ByteVec, targetSequence: U256, coreModule: U256, action: ByteVec) -> (U256, ByteVec) {
+    pub fn parseAndVerifyGovernanceVAAGeneric(vaa: ByteVec, targetSequence: U256, coreModule: U256, action: ByteVec) -> (U256, U256, ByteVec) {
         let (emitterChainId, targetChainId, emitterAddress, msgSequence, payload) = parseAndVerifyVAA(vaa, true)
         assert!(emitterChainId == governanceChainId, ErrorCodes.InvalidEmitChainId)
         assert!(emitterAddress == governanceEmitterAddress, ErrorCodes.InvalidEmitAddress)
-        assert!(msgSequence == targetSequence, ErrorCodes.InvalidSequence)
+        assert!(msgSequence >= targetSequence, ErrorCodes.InvalidSequence)
```

Comment:
This is because the `Governance` contract and the `TokenBridge` contract use the same `GovernanceEmitterChainId` and `GovernanceEmitterAddress`, it is possible that the two vaas acting on the `Governance` contract and the `TokenBridge` contract respectively have the same vaa id, which will cause storage problems.

All governance messages are generated by off-chain guardians, so only +2/3 guardians signing a vaa with a large msgSequence would cause an attack.